nicegui/run.py

```diff
@@ -21,9 +26,10 @@ async def _run(executor: Any, callback: Callable, *args: Any, **kwargs: Any) ->
             raise
     except asyncio.exceptions.CancelledError:
         pass
+    return cast(R, None)  # the assumption is that the user's code no longer cares that this is a lie
```

Comment:
i don't actually know what the intent of just ignoring CancelledError was, but i'm assuming it's similar to "the app is stopping" case above. At least, it seems unlikely that the intent is that you might occasionally get `None` when awaiting `run.x_bound` ?

Reply:
> i don't actually know what the intent of just ignoring CancelledError was, but i'm assuming it's similar to "the app is stopping" case above.

I think so too.